bin/obj2gltf.js

```diff
@@ -119,29 +131,16 @@ if (!defined(gltfPath)) {
     gltfPath = path.join(path.dirname(objPath), modelName + extension);
 }
 
-var options = {
-    binary : argv.binary,
-    separate : argv.separate,
-    separateTextures : argv.separateTextures,
-    compress : argv.compress,
-    optimize : argv.optimize,
-    optimizeForCesium : argv.optimizeForCesium,
-    generateNormals : argv.generateNormals,
-    ao : argv.ao,
-    kmc : argv.kmc,
-    bypassPipeline : argv.bypassPipeline,
-    checkTransparency : argv.checkTransparency,
-    secure : argv.secure,
-    inputUpAxis : argv.inputUpAxis,
-    outputUpAxis : argv.outputUpAxis
-};
-
 console.time('Total');
 
-obj2gltf(objPath, gltfPath, options)
-    .then(function() {
-        console.timeEnd('Total');
-    })
-    .catch(function(error) {
-        console.log(error.message);
-    });
+try {
+    obj2gltf(objPath, gltfPath, argv)
+        .then(function() {
+            console.timeEnd('Total');
+        })
+        .catch(function(error) {
+            console.log(error.message);
```

Comment:
If they are really DeveloperErrors, they shouldn't be `Project.rejected` and they shouldn't be caught either. DeveloperErrors should never happen as a matter of expected program operation.

If you're telling me that just running obj2gltf may produce DeveloperErrors due to bad input data, then that's not a DeveloperError and should be changed (perhaps on the fly) to a RuntimeError instead.